### PR TITLE
Adding Modal to profile page 

### PR DESCRIPTION
**Pull request by dboston528**
Added modal to confirm delete user

### CODE REVIEW

File: src/ProfilesPage/ProfilePage.js [hunk 1]
```diff
@@ -22,14 +22,26 @@ import UserInfo from "./components/UserInfo";
...
 import EditPage from "../EditPage/EditPage";
+import Modal from 'react-responsive-modal';
+
 
 const userId = sessionStorage.getItem("userId");
 const adminUser = sessionStorage.getItem("adminUser");
 const admin = sessionStorage.getItem("isAdmin");
 // var editButtonClicked = false;
 
+const modStyle = {
```
Comment (mkirlin) [inline src/ProfilesPage/ProfilePage.js]: This doesn't appear to be used anywhere. Can we get rid of it?

File: src/ProfilesPage/ProfilePage.js [hunk 2]
```diff
@@ -262,6 +291,21 @@ class ProfilePage extends Component {
             {this.renderEditUserButton()}
             {this.suspendUserButton()}
             {this.deleteUserButton()}
+            {deleteModalVisible && <Modal style={{borderRadius:"50px"}} open={this.state.deleteModalVisible} onClose={this.onCloseModal} center>
+          <h2 className='text-center'>Are You Sure?</h2>
+          <hr/>
+          <div className='container'>
+          <div className='row'>
+          <div className='col-md-6'>
+          <button className="btn btn-danger" onClick={this.onDelete}>Yup</button>
```
Comment (mkirlin) [inline src/ProfilesPage/ProfilePage.js]: Let's change these to just "Yes" and "No".

File: src/ProfilesPage/ProfilePage.js [hunk 3]
```diff
@@ -164,14 +186,18 @@ class ProfilePage extends Component {
     );
   }
 
+  onCloseModal = () => {
+    this.setState({ deleteModalVisible: false });
+  };
+
+
   deleteUserButton() {
+    console.log(this.state)
```
[the remark this answers is not in the document]
Comment (dboston528) [reply src/ProfilesPage/ProfilePage.js]: For some reason, I don't see this in my code.  I do see `console.log(this.state.editUserEnable);` that I don't remember adding. Should I get rid of that instead?

File: src/ProfilesPage/ProfilePage.js [hunk 4]
```diff
@@ -120,9 +135,16 @@ class ProfilePage extends Component {
     this.setState({ suspendedState: false });
   }
 
-  onDelete() {
+  // @TODO What are we deleting? Please be more specific naming functions
+  onDelete = () => {
     const id = this.props.match.params.id || sessionStorage.getItem("userId");
     this.props.deleteUser(id);
+    window.location.href='/message';
```
[the remark this answers is not in the document]
Comment (dboston528) [reply src/ProfilesPage/ProfilePage.js]: Am I just replacing `window.location.href` with `this.props.history.push` ?

File: src/ProfilesPage/ProfilePage.js [hunk 5]
```diff
@@ -164,14 +186,18 @@ class ProfilePage extends Component {
     );
   }
 
+  onCloseModal = () => {
```
[the remark this answers is not in the document]
Comment (dboston528) [reply src/ProfilesPage/ProfilePage.js]: I made the change.

File: src/ProfilesPage/ProfilePage.js (hunk 1 above)
Comment (dboston528) [reply src/ProfilesPage/ProfilePage.js]: I got rid of it. I think I was trying to add some style to the modal.

File: src/ProfilesPage/ProfilePage.js (hunk 2 above)
Comment (dboston528) [reply src/ProfilesPage/ProfilePage.js]: I changed it.

File: src/ProfilesPage/ProfilePage.js [hunk 6]
```diff
@@ -120,9 +135,16 @@ class ProfilePage extends Component {
     this.setState({ suspendedState: false });
   }
 
-  onDelete() {
+  // @TODO What are we deleting? Please be more specific naming functions
+  onDelete = () => {
```
[the remark this answers is not in the document]
Comment (dboston528) [reply src/ProfilesPage/ProfilePage.js]: Should I change the name to something more specific, or just leave it as is?

File: src/ProfilesPage/ProfilePage.js [hunk 7]
```diff
@@ -111,6 +125,7 @@ class ProfilePage extends Component {
     this.setState({ suspendedState: true });
   }
 
+  // @TODO onUnsuspend what? Please be more specific naming functions
```
[the remark this answers is not in the document]
Comment (dboston528) [reply src/ProfilesPage/ProfilePage.js]: Should I change the name to something more specific, or just leave it as is?

File: src/ProfilesPage/ProfilePage.js [hunk 8]
```diff
@@ -103,6 +116,7 @@ class ProfilePage extends Component {
     this.setState({ editUserEnable: editable });
   }
 
+  // @TODO onSuspend what? Please be more specific naming functions
```
[the remark this answers is not in the document]
Comment (dboston528) [reply src/ProfilesPage/ProfilePage.js]: Should I change the name to something more specific, or just leave it as is?

**Pull request merged**